### DISCUSSION TITLE
Vienna for iOS?

### DISCUSSION THREAD

**Discussion by mariano-daniel** [ViennaRSS/vienna-rss#1838] [Ideas]
We **definitely** need it and I at least would pay for it!

**Comment by josh64x2** [comment]
This is a fun idea, however we don't have the capacity right now to work on this. But let us think about it 😄 :

- What are the current options for rss readers on iOS?
- What would make Vienna unique?
- How would the current user interface translate to iOS?

Let me know what you think so maybe one day we have some ideas to implement.

**Reply by mariano-daniel** [reply]
Thank you! Replying to your questions:

* Current options in iOS:
![image](https://github.com/user-attachments/assets/c5ff1e9e-f595-44ff-bf54-075159b55b49)
![image](https://github.com/user-attachments/assets/195d9c60-892f-46ae-a3b1-492c3228b92f)

* Vienna **is** and will always be unique to us! <3 But the idea of transporting my RSS feeds from laptop to phone sounds like something I'm not seeing a lot on iOS (unless you download Vivaldi Browser, which sounds overkill just to read news) . I like being able to read my news in RSS without having to clutter my browser with tabs, etc.

* Not sure if you noticed, but one of the discussions here on GitHub reads "Don't ever change the UI".  I just installed Vienna on MacOS Monterey and it's BEAUTIFUL. The UI is indeed perfect and I wouldn't add or remove anything at all! Actually, I'm browsing the Vienna website to see how can I chip in, ko-fee or I can even contribute to localize it in Spanish if you like!

**Reply by barijaona** [reply]
> * What are the current options for rss readers on iOS?

I'd add those that I discovered by answering to another thread:

- [Reeder Classic](https://www.reederapp.com/classic/) (paid)
- [Fluent Reader Lite](https://hyliu.me/fluent-reader-lite/) (free)
- [Fiery Feeds](https://voidstern.net/fiery-feeds) (free, premium subscription is not necessary)
- [NetNewsWire](https://netnewswire.com) (free, open source)
- [ReadKit](https://readkit.app) (free, premium subscription is not necessary)

They sync with Vienna through a OpenReader server (like FreshRSS).
